composer.json

```diff
@@ -25,7 +25,8 @@
         "php": ">=8.1",
         "cakephp/cache": "^5.2",
         "cakephp/orm": "^5.2",
-        "robmorgan/phinx": "^0.16.10"
+        "symfony/config": "^6.0 || ^7.0",
+        "symfony/console": "^6.0 || ^7.0"
```

Comment:
We should be able to remove these dependencies as well when some more code is removed.